components/InteriorLeftNavItem.js

```diff
@@ -40,9 +46,7 @@ const InteriorLeftNavItem = ({
       onKeyPress={evt => onClick(evt, href)}
       {...other}
     >
-      <a className="left-nav-list__item-link" href={href} tabIndex={tabIndex}>
-        {label}
-      </a>
+      {newChild}
```

Comment:
That's what I'm worried about. 
This is more of a reverting back to an [older version of this component](https://github.com/carbon-design-system/carbon-components-react/blob/7e195d77fe79109a86ef2d61f1c4ab08bb9de5a6/components/InteriorLeftNavItem.js#L46).

- It would be breaking for people who  have already adapted to using the current InteriorLeftNav (ILN).
- But right now this current version doesn't work for people who need to pass in single `children` for use with React Router's `<Link>` component.

Is there a better workaround for this?